pkgs/top-level/all-packages.nix

```diff
@@ -132,6 +132,21 @@ with pkgs;
       ../build-support/setup-hooks/autoreconf.sh
   ) { };
 
+  sphinxHook =
+    let opts = {
+          name = "sphinx-hook";
+          deps = [
+            findutils
+            python3.pkgs.sphinx
+            texinfo
+            # This is necessary to generate pdf.
+            texlive.combined.scheme-full
```

Comment:
scheme-full is huge. Is there a smaller variant that works?

Reply:
Afaik even `scheme-small` comes with pdflatex.

Reply:
Yep. It also needs latexmk, cmap and tg-something (don't recall name of the package). I spent probably hour trying to cherry-pick right subset of packages, but eventually gave up and put schema-full.